pkg/agent/flowexporter/connections/connections_test.go

```diff
@@ -15,7 +15,6 @@
 package connections
```

Comment:
should the file name be conntrack_connections_test.go?

Reply:
This file tests the methods of struct `connectionStore`: methods from both conntrackConnectionStore and denyConnectionStore are tested. I didn't use separate files because much of the methods are shared. Do you think it's better to divide into two testing files?

Reply:
Yes, keeping three files makes it clearer. Updated.